actors/common/src/main/java/org/sunbird/learner/actors/user/service/UserService.java

```diff
@@ -64,4 +57,12 @@ public void checkKeyUniqueness(String key, String value, boolean isEncrypted) {
     }
   }
 
+  public CassandraOperation getCassandraOperation(){

```

Comment:
getCassandraOperation method is called by only this class , so scope can be private  
same for getEncryptionService

Reply:
done